components/builder-api/src/server/services/memcache.rs

```diff
@@ -206,6 +210,35 @@ impl MemcacheClient {
         ret
     }
 
+    pub fn get_origin_member_role(&mut self, origin: &str, account_id: u64) -> Option<String> {
+        trace!("Getting origin role membership for {} {} from memcached",
+               origin,
+               account_id);
+
+        let key = format!("member_role:{}/{}", origin, account_id);
```

Comment:
Why doesn't this use member_role_ns_key?

Reply:
see above